src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/OffenderServiceTest.kt

```diff
@@ -671,22 +671,6 @@ class OffenderServiceTest {
     assertThat(offenderService.isLao(offenderDetails.otherIds.crn)).isFalse
   }
 
-  @Test
-  fun `returns false and parse getUserAccessForOffenderCrn response successfully by ignoring unexpected element in the 403 response`() {
-    val crn = "ABC123"
-    val deliusUsername = "USER"
-    every { mockOffenderDetailsDataSource.getUserAccessForOffenderCrn(deliusUsername, crn) } returns StatusCode(
-      status = HttpStatus.FORBIDDEN,
-      method = HttpMethod.GET,
-      path = "/secure/offenders/crn/$crn/user/$deliusUsername/userAccess",
-      body = "{\"userRestricted\":false,\"userExcluded\":true,\"exclusionMessage\":\"You are excluded\"}",
-    )
-
-    val result = offenderService.canAccessOffender(deliusUsername, crn)
-
-    assertThat(result).isFalse()
-  }
```

Comment:
should we not have unit tests for the new function?

Reply:
This is for the old function when it return ClientResult.Failure.StatusCode and HttpStatus.FORBIDDEN is not needed when using delius

Reply:
I agree this needs removing but we have a new function that doesn't appear to have unit tests? 

``` fun canAccessOffenders(username: String, crns: List<String>): Map<String, Boolean> ```

Reply:
add unitests